### ISSUE TITLE
just Just produced lib file

### ISSUE THREAD

**Issue by ctcommon** [Microsoft/DirectXTex#88]
I use VS2017 with win10,  I tried 2017_win10.sln and 2017.sln,but it just produced DirectXTex.lib. I can't find the Texassemble.exe.

**Comment by walbourn** [comment]
Both ``DirectXTex_Desktop_2015_Win10.vcxproj`` and ``DirectXTex_Desktop_2017_Win10.vcxproj`` use ``_WIN32_WINNT=0x0A00`` which means the resulting binary only works on Windows 10. I do this in DirectXTex primarily to add DirectX 12 API support which also requires Windows 10.

The command-line utilities are designed to work down-level on Windows 7 and don't use DirectX12 APIs. As such, they are only built with the  ``DirectXTex_Desktop_2015.vcxproj`` and ``DirectXTex_Desktop_2017.vcxproj`` projects which use ``_WIN7_PLATFORM_UPDATE`` / ``_WIN32_WINNT=0x0600``.

This keeps the number of projects I have to maintain down (2 per command-line tool).

**TL;DR:** Use ``DirectXTex_Desktop_2017.sln`` to build command-line tools.
